antora.yml

```diff
@@ -15,11 +17,11 @@ asciidoc:
     # Fallback versions
     # We try to fetch the latest from GitHub at build time
     # --
-    full-version: 24.2.2
+    full-version: 24.3.1
     latest-release-commit: '72ba3d3'
     latest-operator-version: 'v2.2.0-24.2.2'
     latest-redpanda-helm-chart-version: 5.8.3
-    redpanda-beta-version: 24.2.2-rc5
+    redpanda-beta-version: 24.3.1-rc1
```

Comment:
RC5 is now the latest

Reply:
this is not necessary anymore as this value is auto-fetched from github. That's just the fallback version. The reason this is showing up as diff is simply because this PR is old and should be rebased.